deploy/99_ustStrategy.ts

```diff
@@ -84,7 +84,12 @@ const func = async function (env: HardhatRuntimeEnvironment) {
   });
 
   await mockEthAnchorRouter.addPendingOperator(ethAnchorOperator1);
-  await vault.connect(owner).updateInvested("0x");
+  await (await vault.connect(owner).updateInvested("0x")).wait();
```

Comment:
await await wait? wait what? xD

Reply:
need to await updateInvested so the script doesn't continue before the transaction is submitted

wait() is to wait for the transaction to be mined, and it's an async function so I need to await that so the script doesn't continue until that's done

like that the notifDepositResult command after doesn't error because the updateInvested transaction was not mined yet

bit of an eyesore but only alternative is to split over multiple lines